crates/bevy_input/src/mouse/event.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{
+    mouse::{MouseButton, MouseScrollUnit},
+    ElementState,
+};
+use bevy_math::Vec2;
+
+/// A mouse button input event
+#[derive(Debug, Clone)]
+pub struct MouseButtonInput {
+    pub button: MouseButton,
+    pub state: ElementState,
+}
+
+/// A mouse motion event
+#[derive(Debug, Clone)]
+pub struct MouseMotion {
+    pub delta: Vec2,
+}
+
+/// A mouse scroll wheel event, where x represents horizontal scroll and y represents vertical
+/// scroll.
+#[derive(Debug, Clone)]
+pub struct MouseWheel {
```

Comment:
This belongs in scrolling.rs now :)

Reply:
I don't agree. I like it being inside of `event.rs` since all of them are events. That's also the way they are structured inside of the other PRs. For me it seems more intuitive to search for an event in a file called `event.rs` instead of having to guess where it might be. I definitely get your point though. 

The question is: Do we want to organize things by `type of thing` or by `context of thing`?

`type of thing`: E.g. all of the events.
`context of thing`: E.g. all of the things that have to do with scrolling.

I'm definitely in favor of `type of thing`.

Reply:
I prefer context of thing: over time, these structs tend to get interwoven.